SELF-ASSESMENT.md

```diff
@@ -44,3 +56,359 @@ See [README](https://github.com/jaegertracing/jaeger/#jaeger---a-distributed-tra
+
+#### Vulnerability Response Process.
+
+_Who is responsible for responding to a report. What is the reporting process? How would you respond?_
+
+A report can be sent via email at jaeger-tracing@googlegroups.com, open an issue on GitHub or send a message to the #jaeger channel on the CNCF Slack encrypted with Jaeger’s public key. Responses can be sent directly via the method used to contact the maintainers by the maintainers.
+
+#### Incident Response.
+
+_A description of the defined procedures for triage, confirmation, notification of vulnerability or security incident, and patching/update availability._
+
+Security Advisories are listed and responded to on the [security tab of the Jaeger GitHub](https://github.com/jaegertracing/jaeger/security/advisories).
+
+## Appendix
+
+### Known Issues Over Time.
+
+_List or summarize statistics of past vulnerabilities with links. If none have been reported, provide data, if any, about your track record in catching issues in code review or automated testing._
+
+#### Miscellaneous Issues
+
+* The Jaeger architecture relies on a central collector to receive and store trace reports from agents and microservices. The collector exposes HTTP endpoints like /api/traces for trace data submission, lacking authentication. This vulnerability opens the possibility of a Server-Side Request Forgery (SSRF) attack, enabling a compromised microservice to submit malicious trace data to the collector. Implementing token-based access enhances collector security by restricting entry to authenticated agents.
+
+### [CII Best Practices](https://www.coreinfrastructure.org/programs/best-practices-program/).
+
+_Best Practices. A brief discussion of where the project is at with respect to CII best practices and what it would need to achieve the badge._
+
+* The Jaeger project has achieved passing level criteria under Open Source Security Foundation Best Practices and is in the process of obtaining the silver badge.
+* Jaeger-client-js and Jaeger-client-javascript is in the process of obtaining passing level criteria.
+
+### Case Studies.
+
+_Provide context for reviewers by detailing 2-3 scenarios of real-world use cases._
+
+#### Ticketmaster:
+
+* Consisting of over 300 microservices, the company benefits from the Jaeger project, utilizing the system to track over 100 million transactions a day. 
+* As the company scaled, they quickly learned that logging was insufficient and bulky. It helped uncomplicate the log aggregation process which generated terabytes of data daily. Jaeger simplified the debugging process by providing visibility into critical areas. – where developers could find root causes of issues and save time doing so.
+* Ticketmaster uses the Directed Acyclic Graph (DAG) view to see request flow. 
+* They also take advantage of Jaeger’s powerful adaptive sampling feature which reduces overhead without reducing visibility into systems.
+
+#### Grafana Labs:
```

Comment:
@joe-elliott I think this was true when published, but is GL still using Jaeger?